test/integration/tcp_tunneling_integration_test.cc

```diff
@@ -87,6 +94,27 @@ TEST_P(ConnectTerminationIntegrationTest, Basic) {
   ASSERT_FALSE(response_->reset());
 }
 
+TEST_P(ConnectTerminationIntegrationTest, UsingHostMatch) {
+  exact_match_ = true;
+  initialize();
+
+  connect_headers_.removePath();
```

Comment:
Was `path` un-needed for other tests too (my understanding is that H/2 may or may not have it, but it is unnecessary)? Just curious in case I'm missing something particular about why this test would remove it.

Reply:
Good spotting - It would be fine to leave in I just wanted to make sure we tested sans path too :-)